From f9ffaa13831a339482c2520eb03e8f9d0d17e3d6 Mon Sep 17 00:00:00 2001
From: Dan Pasanen <dan.pasanen@gmail.com>
Date: Thu, 11 May 2017 10:00:23 -0500
Subject: [PATCH 095/249] cm: Remove Gello

* Jelly is more lightweight and much easier to maintain. Gello can
  live on if someone decides they want to compile it and host it
  somewhere.

Change-Id: Id9f3f6877e15c4081325fe9d34488de2860f8042
---
 snippets/cm.xml | 2 --
 1 file changed, 2 deletions(-)

diff --git a/snippets/cm.xml b/snippets/cm.xml
index a9bf120..837c6e3 100644
--- a/snippets/cm.xml
+++ b/snippets/cm.xml
@@ -9,7 +9,6 @@
   <project path="external/exfat" name="LineageOS/android_external_exfat" />
   <project path="external/ffmpeg" name="LineageOS/android_external_ffmpeg" />
   <project path="external/fuse" name="LineageOS/android_external_fuse" />
-  <project path="external/gello-build" name="LineageOS/android_external_gello_build" />
   <project path="external/htop" name="LineageOS/android_external_htop" />
   <project path="external/koush/ion" name="LineageOS/ion" />
   <project path="external/libncurses" name="LineageOS/android_external_libncurses" />
@@ -43,7 +42,6 @@
   <project path="packages/apps/Eleven" name="LineageOS/android_packages_apps_Eleven" />
   <project path="packages/apps/Exchange" name="LineageOS/android_packages_apps_Exchange" />
   <project path="packages/apps/FMRadio" name="LineageOS/android_packages_apps_FMRadio" />
-  <project path="packages/apps/Gello" name="LineageOS/android_packages_apps_Gello" />
   <project path="packages/apps/Jelly" name="LineageOS/android_packages_apps_Jelly" />
   <project path="packages/apps/LockClock" name="LineageOS/android_packages_apps_LockClock" />
   <project path="packages/apps/Profiles" name="LineageOS/android_packages_apps_Profiles" />
-- 
2.17.1

